addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -387,7 +387,7 @@
                 </div>
                 <div class="col-8">
                     <h3 class="w-100" t-att-title="slide_promoted.name">
-                        <t t-if="slide_promoted_access['can_access']">
+                        <t t-if="slide_promoted.is_preview or channel.is_member or is_slides_publisher">
```

Comment:
Could even use channel.can_publish here and get rid og this "is_slides_publisher" thing imo.